package.json

```
@@ -1,32 +1,48 @@
 {
   "name": "serverless-log-forwarding",
-  "version": "2.0.0",
+  "version": "3.0.0",
+  "engines": {
+    "node": ">=14"
```

```suggestion
    "node": ">=12"
```

https://nodejs.org/en/about/releases/
The end of life is 2022-04-30 for node 12 so it's better to support for now

---

addressed